demo/src/index.js

```diff
@@ -69,6 +97,15 @@ class Demo extends Component {
           brushColor="rgba(155,12,60,0.3)"
           imgSrc="https://upload.wikimedia.org/wikipedia/commons/a/a1/Nepalese_Mhapuja_Mandala.jpg"
         />
+
+        <h2>Refreshable Background Image</h2>
+        <p>You can also set the `refreshBackgroundImage` prop to true is you want to enable background refreshable.</p>
+        <p>This will refresh the background in every two seconds.</p>
+        <CanvasDraw
+          brushColor="rgba(155,12,60,0.3)"
+          refreshBackgroundImage={true}
+          imgSrc={this.state.backgroundImg}
+        />
```

Comment:
Based on another comment, we would then loose the `refreshBackgroundImage` prop, and update every time the imgSrc changes.

```suggestion
        <h2>Refreshable Background Image (New)</h2>
        <p>You can also update the background image and it will be updated in the Canvas.</p>
        <CanvasDraw
          brushColor="rgba(155,12,60,0.3)"
          imgSrc={this.state.backgroundImg}
        />
```